addon-test-support/@ember/test-helpers/dom/-logging.ts

```diff
@@ -0,0 +1,65 @@
+import Target from './-target';
+
+/**
+ * Logs a debug message to the console if the `testHelperLogging` query
+ * parameter is set.
+ *
+ * @private
```

Comment:
FWIW, I wonder if this should actually be private. Seems like it might be nice for folks authoring custom helpers too...

Reply:
I'd say let's try it out and see if it works properly before we commit to making it a public API